.tekton/odh-kuberay-operator-controller-push.yaml

```diff
@@ -7,7 +7,7 @@ metadata:
     build.appstudio.redhat.com/target_branch: '{{target_branch}}'
     pipelinesascode.tekton.dev/cancel-in-progress: "false"
     pipelinesascode.tekton.dev/max-keep-runs: "3"
-    pipelinesascode.tekton.dev/on-cel-expression: event == "push" && target_branch == "release-v1.4.0"
+    pipelinesascode.tekton.dev/on-cel-expression: event == "push" && target_branch == "dev"
```

Comment:
should this be `release-v1.4.2` or `dev`?

Reply:
we shouldn't have a release-v1.4.2 branch anymore, so dev is the suitable branch.